core/src/validator.rs

```diff
@@ -610,6 +610,12 @@ impl Validator {
         tpu_config: ValidatorTpuConfig,
         admin_rpc_service_post_init: Arc<RwLock<Option<AdminRpcRequestMetadataPostInit>>>,
     ) -> Result<Self> {
+        #[cfg(debug_assertions)]
+        const DEBUG_ASSERTION_STATUS: &'static str = "enabled";
+        #[cfg(not(debug_assertions))]
+        const DEBUG_ASSERTION_STATUS: &'static str = "disabled";
+        info!("debug-assertion support: {DEBUG_ASSERTION_STATUS}");
```

Comment:
why here instead of, say, `validator/main.rs`?

This will log even more redundant stuff for local cluster tests, and I think we only really care about this for non-test scenarios

Reply:
debug-assertion support: --> debug-assertion status:
I think it is probably okay to log it as it is only logged once at validator start